tests/bluetooth/iso/modules/iso_broadcast_src.c

```diff
@@ -117,7 +117,7 @@ static void broadcaster_t(void *arg1, void *arg2, void *arg3)
 	while (1) {
 		int ret;
 
-		k_sleep(K_USEC(big_create_param.interval));
+		k_sleep(K_USEC(big_create_param.interval - 200));
```

Comment:
Seems random to reduce by 200. Add a comment?

Reply:
Will add it, this part is very similar to here:
https://github.com/zephyrproject-rtos/zephyr/blob/b3ffe0aede6db181e118eb95db84ad7537bed0b2/samples/bluetooth/iso_broadcast_benchmark/src/broadcaster.c#L583